Client/Frontend/Reader/ReaderModeHandlers.swift

```diff
@@ -44,14 +45,10 @@ struct ReaderModeHandlers {
                             }
                         }
                         
-                        // Must generate a unique nonce, every single time as per Content-Policy spec.
-                        let setTitleNonce = UUID().uuidString.replacingOccurrences(of: "-", with: "")
-                        
-                        if let html = ReaderModeUtils.generateReaderContent(readabilityResult, initialStyle: readerModeStyle,
-                            titleNonce: setTitleNonce),
+                        if let html = ReaderModeUtils.generateReaderContent(readabilityResult, initialStyle: readerModeStyle),
                             let response = GCDWebServerDataResponse(html: html) {
                             // Apply a Content Security Policy that disallows everything except images from anywhere and fonts and css from our internal server
-                            response.setValue("default-src 'none'; img-src *; style-src http://localhost:* '\(readerModeStyleHash)'; font-src http://localhost:*; script-src 'nonce-\(setTitleNonce)'", forAdditionalHeader: "Content-Security-Policy")
+                            response.setValue("default-src 'none'; img-src *; style-src http://localhost:* '\(readerModeStyleHash)'; font-src http://localhost:*; script-src '\(readerModeScriptHash)'", forAdditionalHeader: "Content-Security-Policy")
```

Comment:
When ran live, the dynamic script won't run as the hash is static and the content of the script is replaced at runtime.

Looking into putting the JS in a separate file atm.